client/src/pages/Login.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useContext } from 'react'
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core/styles';
+import * as imageURL from '../images/login-photo.png'
+import { Link } from 'react-router-dom'
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+      width: '25ch',
+      length: '400ch',
+      display: 'flex',
+      flexDirection: 'column',
+      marginBottom: '30px'
+    }
+  },
+}));
+
+const ContinueButton = withStyles({
+  root: {
+    background: 'linear-gradient(45deg, rgba(73, 94, 203, .8) 30%, rgba(73, 145, 203, .8) 90%)',
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+    boxShadow: '0 3px 5px 2px rgba(125, 123, 135, .3)',
+    marginLeft: '10px'
+  },
+})(Button);
+
+const SignupButton = withStyles({
+  root: {
+    background: 'linear-gradient(45deg, rgba(105, 105, 105, .8) 30%, rgba(169, 169, 169, .8) 90%)',
```

Comment:
You should be adding these colors to your theme.js file - this will allow us to reuse these colors across multiple components and also for all buttons